From e1ae78ff914d4a7cb79f663ced0e57b67f6018e1 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 14 Jun 2022 09:34:54 +0800
Subject: [PATCH 7/7] Revert "drm/amd/display/dc/gpio/gpio_service: Pass around
 correct dce_{version, environment} types"

This reverts commit f89f76f4b0e7ae7d7f5316fe0584c4cb79fa2507.
---
 drivers/gpu/drm/amd/display/dc/gpio/gpio_service.c   | 12 ++++++------
 .../drm/amd/display/include/gpio_service_interface.h |  4 ++--
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/gpio/gpio_service.c b/drivers/gpu/drm/amd/display/dc/gpio/gpio_service.c
index dae8e489c8cf..92280cc05e2d 100644
--- a/drivers/gpu/drm/amd/display/dc/gpio/gpio_service.c
+++ b/drivers/gpu/drm/amd/display/dc/gpio/gpio_service.c
@@ -53,8 +53,8 @@
  */
 
 struct gpio_service *dal_gpio_service_create(
-	enum dce_version dce_version,
-	enum dce_environment dce_environment,
+	enum dce_version dce_version_major,
+	enum dce_version dce_version_minor,
 	struct dc_context *ctx)
 {
 	struct gpio_service *service;
@@ -67,14 +67,14 @@ struct gpio_service *dal_gpio_service_create(
 		return NULL;
 	}
 
-	if (!dal_hw_translate_init(&service->translate, dce_version,
-			dce_environment)) {
+	if (!dal_hw_translate_init(&service->translate, dce_version_major,
+			dce_version_minor)) {
 		BREAK_TO_DEBUGGER();
 		goto failure_1;
 	}
 
-	if (!dal_hw_factory_init(&service->factory, dce_version,
-			dce_environment)) {
+	if (!dal_hw_factory_init(&service->factory, dce_version_major,
+			dce_version_minor)) {
 		BREAK_TO_DEBUGGER();
 		goto failure_1;
 	}
diff --git a/drivers/gpu/drm/amd/display/include/gpio_service_interface.h b/drivers/gpu/drm/amd/display/include/gpio_service_interface.h
index 7e3240e73c1f..9c55d247227e 100644
--- a/drivers/gpu/drm/amd/display/include/gpio_service_interface.h
+++ b/drivers/gpu/drm/amd/display/include/gpio_service_interface.h
@@ -42,8 +42,8 @@ void dal_gpio_destroy(
 	struct gpio **ptr);
 
 struct gpio_service *dal_gpio_service_create(
-	enum dce_version dce_version,
-	enum dce_environment dce_environment,
+	enum dce_version dce_version_major,
+	enum dce_version dce_version_minor,
 	struct dc_context *ctx);
 
 struct gpio *dal_gpio_service_create_irq(
-- 
2.27.0

